### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 10th August 15:00UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1046]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - Stressnet hardfork this Saturday https://github.com/spackle-xmr/monero/releases/tag/v251.18.3.3.0
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/)    
  a. [New Monero Website](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/450)  re-opened after monero-site [workgroup meeting](https://libera.monerologs.net/monero-site/20240728#c404275)  
  b. [ofrnxmr feat. BasicSwapDEX - take over the world pt 2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457)    
  c. [Monero Poker Infrastructure Project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/461)  
  d. [VOSTOEMISIO - FCMP Animated Explainer Video & Remake Introductory Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/472)    Combined into 1 proposal with 2 milestones after positive merge sentiment  
  e. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477)   Updated 25>35 hrs per week per 2 devs after an [update](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477#note_25561) Maker GUI + ASB improvements
  f. [yamabiiko full-time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/479)    closed due to personal / availability reasons
  g. [Part-time + Flexible Work on getmonero.org (1 Month)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/481)    hardenedsteel
  h. [ofrnxmr support, docs, site, meta++ (totw pt3)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/482)    
  i. [selsta part-time monero development (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/483) 
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2024](https://monerokon.org)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/1039)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
> __< plowsof >__ Meeting in 1hr25~mins https://github.com/monero-project/meta/issues/1046     

> __< plowsof >__ Meeting time https://github.com/monero-project/meta/issues/1046     

> __< plowsof >__ Greetings . Stressnet hardfork is today, please dont forget to run or update your stressnet node :)      

> __< binarybaron >__ hey everyone     

> __< c​hch3003:monero.social >__ Hey     

> __< n​ihilist:m.datura.network >__ ayy     

> __< plowsof >__ Cypherstack has  released a report for the FCMP++ project @ 'Divisors report' comment at the bottom https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/449     

> __< h​into:monero.social >__ hello     

> __< o​frnxmr:monero.social >__ Morning     

> __< r​ucknium:monero.social >__ spackle's report on the first two months of stressnet: https://reddit.com/r/Monero/comments/1eoana8/the_stressnet_so_far/     

> __< plowsof >__ nice     

> __< o​frnxmr:monero.social >__ https://github.com/spackle-xmr/monero     

> __< o​frnxmr:monero.social >__ Instructions to join the stressnet are in the readme. If you can run 2 nodes, the more the merrier     

> __< plowsof >__ other news sources: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [Monero Moon](https://www.themoneromoon.com/)      

> __< plowsof >__ vthor is almost ready to push a production ready version of the xmr seed signer (work in porgress ccs https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/465)      

> __< plowsof >__ we can move on to the open ccs ideas unless there are other open topics/highlights people wish to mention      

> __< plowsof >__ ah, monero-docs workgroup has made some nice progress towards an official docs site for getmonero :) #monero-docs     

> __< plowsof >__ ok, lets move on to the ccs merge list     

> __< plowsof >__   a. [New Monero Website](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/450)         

> __< o​frnxmr:monero.social >__ Should be live in the next 2 weeks or so, albeit needs updating of content     

> __< o​frnxmr:monero.social >__ (docs)     

> __< plowsof >__ (docs) https://github.com/monero-project/monero-docs (test branch currently)      

> __< plowsof >__ sorry, one more highlight, rbrunner has moved NWLB meetings into #monero-dev :) https://github.com/monero-project/meta/issues/1053      

> __< o​frnxmr:monero.social >__ The website ccs is largely stalled pending a redesign by Diego Salazar:  and vostoemisio:  .... idk whats going on     

> __< o​frnxmr:monero.social >__ another -site meeting tomorrow i guess     

> __< plowsof >__ yes, the ccs is supposodly going to change or be acompanied by a re-design proposal by diego (who is currently busy at defcon) https://github.com/monero-project/meta/issues/1049      

> __< plowsof >__ but open to feedback if anyone wishes to share something now, otherwise we can move on      

> __< d​iego:cypherstack.com >__ Hi I'm here. :)     

> __< d​iego:cypherstack.com >__ My apologies for my delays on this.     

> __< plowsof >__ hello diego      

> __< d​iego:cypherstack.com >__ At defcon and other things the past couple of weeks have required my attention.     

> __< plowsof >__ more details on the changes to the ccs will be confirmed tomorrow in the site meeting?     

> __< o​frnxmr:monero.social >__ Likely not, diego busy and vost hasnt looked into it yet     

> __< c​hch3003:monero.social >__ ...     

> __< o​frnxmr:monero.social >__ Weird politics 🤷‍♂️, i dont know whats wrong w current design     

> __< d​iego:cypherstack.com >__ Would you to take a stab at the redesign janaka?     

> __< plowsof >__ janaka needs a designer, the designer is presumably going to be diego      

> __< d​iego:cypherstack.com >__ I don't HAVE to do it     

> __< d​iego:cypherstack.com >__ Someone else can do it.     

> __< c​hch3003:monero.social >__ I am not a designer     

> __< d​iego:cypherstack.com >__ But the proposed design is not good.     

> __< plowsof >__ janaka not being a designer and we have one with highly relavant experience      

> __< c​hch3003:monero.social >__ I don't master Figma enough     

> __< d​iego:cypherstack.com >__ I'm sorry for the frustration.     

> __< c​hch3003:monero.social >__ As I understand Diego want to keep the structure of the current website, so it would require a complete redesign     

> __< d​iego:cypherstack.com >__ Again, these are the things I am proposing and others can disagree.     

> __< plowsof >__ if the designer proposal is going to be separate then janakas work/proposal is not contingent on a design proposal being ironed out?       

> __< d​iego:cypherstack.com >__ We can discuss that at tomorrow's meeting for the site.     

> __< d​iego:cypherstack.com >__ No. I have an active CCS proposal.     

> __< d​iego:cypherstack.com >__ My hours redesigning would presumably be covered.     

> __< plowsof >__ page 166, section 5: multiple ccs' are fine. citation mjxmr      

> __< plowsof >__ ok nice, i think we can move on and return after sites meeting tomorrow      

> __< plowsof >__   b. [ofrnxmr feat. BasicSwapDEX - take over the world pt 2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457)         

> __< plowsof >__ are we skipping this proposal ofrnxmr in favour of discussing your latest one?     

> __< plowsof >__ later on     

> __< o​frnxmr:monero.social >__ Made adjustments to the milestones (lowered amounts, split up, adjusted to 160/xmr rate)     

> __< plowsof >__ thanks     

> __< o​frnxmr:monero.social >__ Vote spread is 24:9 or so, up from 20:10     

> __< plowsof >__ couple of files in the commit but we can ignore that for now      

> __< o​frnxmr:monero.social >__ merged master into the branch, but u can see diffs on the compare latest version screen     

> __< plowsof >__ also thanks janaka for sprinkling votes on the open ideas, much appreciated      

> __< vthor >__ vthor is almost ready to push a production ready version of the xmr seed signer (work in porgress ccs)...     

> __< plowsof >__ :D     

> __*__ vthor is jumping of a bridge (which doesn't even exists)     

> __< c​hch3003:monero.social >__ Haha you are welcome plowsof     

> __< plowsof >__ thank you for the update vthor, great work.      

> __< vthor >__ :D     

> __< plowsof >__ ofrnxmr setting a record for proposal interaction / updoots perhaps, we can move on unless ... also basicswapdex team still want to solve the multisig ux quickly for us, in a seperate proposal, perhaps related to this or the other or both ofrnxmr proposals?     

> __< o​frnxmr:monero.social >__ Yeah, the bsx team is looking to do the multisig ux in feather for us     

> __< o​frnxmr:monero.social >__ CryptoGuard:     

> __< plowsof >__ i would like to skip entirely   c. [Monero Poker Infrastructure Project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/461)    , stale / close sentiment , unless there are objections      

> __< plowsof >__  d. [VOSTOEMISIO - FCMP Animated Explainer Video & Remake Introductory Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/472)         

> __< plowsof >__ vosto has already had enough positive sentiment / previous contribs to have the combined proposal merged     

> __< o​frnxmr:monero.social >__ I think it could be a fun project, but since its hard to find consensus, perhaps poker would have better success on kuno     

> __< plowsof >__   e. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477)         

> __< binarybaron >__ einliterflasche and I are here in case you have any questions     

> __< plowsof >__ both binarybaron and einliter attended last weeks meeting also which didnt go ahead      

> __< plowsof >__ checks notes to confirm if sethforprivacy returned to update this one      

> __< plowsof >__ yes, so 14 updoots, no down doots, clearly a merge?      

> __< plowsof >__ i think we can move on unless any questions (there was not any last saturday)      

> __< binarybaron >__ great to hear, looking forward to working on the project :)     

> __< plowsof >__   f. [Part-time + Flexible Work on getmonero.org (1 Month)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/481)         

> __< 0​xfffc:monero.social >__ Congrats binarybaron     

> __< o​frnxmr:monero.social >__ I'm neither for nor against, but think bsx and unstoppable both have great support from community. Bsx ccs has a lot of downvoted because some ppl dont like ofrnxmr     

> __< plowsof >__ from hardenedsteel* the part time + flexible      

> __< o​frnxmr:monero.social >__ f. Part time work = hardenedsteel.      

> __< o​frnxmr:monero.social >__ hardenedsteel is one of monero-sites biggest (and one of the only) contributors     

> __< o​frnxmr:monero.social >__ easy +1     

> __< plowsof >__ im not sure what the + flexible is for (we awarded that to long time serving members to reduce any mental suffering from being *gasp* forced to work in return for pay , how dare he come out the gate with that on his first porposal     

> __< o​frnxmr:monero.social >__ Language barrier     

> __< plowsof >__ paid holidays next?? :D     

> __< o​frnxmr:monero.social >__ He means willing to do more than ccs     

> __< o​frnxmr:monero.social >__ paid holidays were born and died with erciccione     

> __< plowsof >__ jokes aside, hardenedsteel has alot of support already. and at 5 xmr, very cheap for continued help      

> __< r​ucknium:monero.social >__ hardenedsteel helped plowsof and me on a previous version of bchmempool.cash . Converting mempool.cash from BTC to BCH     

> __< plowsof >__ this is correct      

> __< plowsof >__ and was jaded by the previous maintainer for not having any PR's accepted, the situation has improved now      

> __< plowsof >__ i'd like to see (even thouhg the website group leader hasn't lol) HS touch base with janaka about .. well.. astro (basically what i need to do tbh so i know what im going to be contributing to in the near future.... use astro for something, like how syntheticbird created cuprate.org using it.. im sure he has a better idea of what 'it' is now )     

> __< 0​xfffc:monero.social >__ I haven’t read the ccs. I am not against it, and don’t know much to be in favour of it. But never had any negative interactions with hardenedsteel. Looks like someone who wants to work.     

> __< plowsof >__ site workgroup can bring this up tomorrow also hopefully      

> __< plowsof >__ otherwise , im sure he'll be able to make the same contirbutions even with a new site/back end      

> __< o​frnxmr:monero.social >__ Why?     

> __< o​frnxmr:monero.social >__ the workgroup is like 4 ppl and were all here     

> __< plowsof >__ current site is gone, we need help for new site , people ideally who know what astro is      

> __< o​frnxmr:monero.social >__ The -site matrix room has a bunch of ppl who dont contribute to the repo AT ALL     

> __< plowsof >__ or show evidence they can learn well, things      

> __< o​frnxmr:monero.social >__ Dont even look at the prs or issues     

> __< plowsof >__ lol     

> __< plowsof >__ i am supportive/thankful for HS's willing to help/contribute to site, shall we move on      

> __< o​frnxmr:monero.social >__ i dont think their comments on hardened's ccs are helpful at all. Peak larping     

> __< o​frnxmr:monero.social >__ Yea 😅     

> __< plowsof >__   g. [ofrnxmr support, docs, site, meta++ (totw pt3)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/482)         

> __< plowsof >__ ofrnxmrs new proposal      

> __< o​frnxmr:monero.social >__ https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/482#note_25683     

> __< o​frnxmr:monero.social >__ scott left some drool in the comments     

> __< r​ucknium:monero.social >__ ofrnxmr: If there are disagreements about the contents of docs, how will they be resolved?     

> __< o​frnxmr:monero.social >__ Not sure if he did it to get a rise out of me, or if he's really that messed up in the head     

> __< o​frnxmr:monero.social >__ Via issues and prs     

> __< r​ucknium:monero.social >__ That doesn't answer my question. The disagreements can appear in PRs and issues. What is the decisionmaking process?     

> __< o​frnxmr:monero.social >__ There is Monero Docs  and #monero-docs     

> __< o​frnxmr:monero.social >__ The same as -site (?)      

> __< o​frnxmr:monero.social >__ not sire what sort of disagreements youre referring to     

> __< plowsof >__ as ive provided some help for the new -docs effort rucknium i think i can speak to that Q also, it seems, in my eyes, a major issue with getmonero.dev is the new license or copyright , MAGIC + others (everywhere)      

> __< r​ucknium:monero.social >__ There is sometimes disagreement about the contents of getmonero.org, especially about non-cotre eneitities. AFAIK there isn't a good way to resolve those disagreements, so some -site progress is slow. Can we do better with -docs?     

> __< o​frnxmr:monero.social >__ If were documenting a function, it only has one definition     

> __< o​frnxmr:monero.social >__ site is slow because of maintainer     

> __< o​frnxmr:monero.social >__ Its a lot faster now, even though we dont have a maintainer.      

> __< o​frnxmr:monero.social >__ there is a contributing guide for site that is rarely followed     

> __< o​frnxmr:monero.social >__ I believe yes we can do better in docs. Core is not in the way     

> __< plowsof >__ current/proposed docs sites are MIT - the new docs (updated mkdocs) has an attribution plugin, where, authors are listed at the bottom of each entry, so edits on getmonero.org can be lifted and attributed appropriately if needed     

> __< 0​xfffc:monero.social >__ +     

> __< 0​xfffc:monero.social >__ Merge.     

> __< r​ucknium:monero.social >__ I have thumbed-up this proposal in the CCS repo.     

> __< o​frnxmr:monero.social >__ Plowsof and dan (is not the man) have already done some good work prepping the site. We should have cookies (netlify replacement fkr pr previews) running soon, as well as having the theming and i18n stuff in     

> __< plowsof >__ apologies hinto for going over      

> __< o​frnxmr:monero.social >__ s/cookies/coolify     

> __< plowsof >__ the same work that warranted a copyright change to MAGIC + others has been completed, basically      

> __< plowsof >__ + extra due to internationalisation , but i guess that will come later      

> __< plowsof >__ i think we can move on for now as we're over      

> __< plowsof >__   h. [selsta part-time monero development (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/483)         

> __< tobtoht_ >__ merge     

> __< 0​xfffc:monero.social >__ Merge.     

> __< plowsof >__ +1      

> __< r​ucknium:monero.social >__ +1     

> __< o​frnxmr:monero.social >__ merge     

> __< plowsof >__ moving on      

> __< plowsof >__   i. [hinto-janai full-time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/484)         

> __< o​frnxmr:monero.social >__ i hate to be that guy.. (who am i kidding, i dont hate it). But idk about these rates     

> __< o​frnxmr:monero.social >__ I know its the same as his last ccs, but the rates are incredibly inflated imo     

> __< o​frnxmr:monero.social >__ We've seen 140-250k/yr rates from devs. Are we supposed to pay cryptographers 500k?     

> __< plowsof >__ im not sure what overlap here , but yamabiiko who recently closed their ccs due to personal / availability reasons was at 50/hr 40hrs a week for some specific cuprate tasks https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/479     

> __< o​frnxmr:monero.social >__ https://github.com/Cuprate/cuprate/commits?author=hinto-janai&since=2024-05-14&until=2024-06-05     

> __< o​frnxmr:monero.social >__ Downgrade gcc     

> __< o​frnxmr:monero.social >__ readme     

> __< o​frnxmr:monero.social >__ Update license     

> __< o​frnxmr:monero.social >__ Add skeleton directories     

> __< o​frnxmr:monero.social >__ Add issue templates     

> __< plowsof >__ hintos most recent ccs update https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/456 (3 llibraries have been added , boog900)     

> __< o​frnxmr:monero.social >__ Create cargo.toml     

> __< o​frnxmr:monero.social >__ Add skeletons     

> __< o​frnxmr:monero.social >__ Add crates     

> __< o​frnxmr:monero.social >__ etc     

> __< o​frnxmr:monero.social >__ Theres no way this is 12k of work (??)     

> __< plowsof >__ for milestones 2+3, so hintos current WIP ccs is finished      

> __< plowsof >__ they alos wish to take a break before beginning the next one , not sure how long      

> __< o​frnxmr:monero.social >__ My last msg was abt mileston 1     

> __< plowsof >__ "Binary/other RPC interface + other work" is MS3      

> __< o​frnxmr:monero.social >__ the update for m2 and m3 was sent last night. Too early to comment on     

> __< plowsof >__ not sure if binary was mentioned here, hopefully hinto can confirm     

> __< plowsof >__ or boog900     

> __< l​ederstrumpf:matrix.org >__ Heya binarybaron & einliterflasche, I like the direction of your proposal! Wanted to ask if you've tried using Farcaster, and if you've looked into integrating it into the UnstoppableSwapGUI in addition to Comit? I know the only significant activity Farcaster's seen in 2024 is adding Namecoin support, but it's fully functional and solves for instance the bidirectional swapping iss<clipped message>     

> __< l​ederstrumpf:matrix.org >__ ue that's been raised here: https://www.reddit.com/r/Monero/comments/1dz739s/comment/lcfi2zs/     

> __< plowsof >__ lederstrumpf the community has decided that farcaster is a dead dodo / abandonned and te only protocol worth investing in is COMIT     

> __< plowsof >__ is this the correct assumption?     

> __< o​frnxmr:monero.social >__ lederstrumpf:  wasnt a wuestion for me, but we'll probably add in basicswap     

> __< o​frnxmr:monero.social >__ Of nvm, i was thinking noots eth swaps.     

> __< o​frnxmr:monero.social >__ Basicswap has bidirectional swaps     

> __< l​ederstrumpf:matrix.org >__ @plowsof: I wanted to leave some extended comments on that on the MR, but seems my account has been deactivated. Who would I get in touch with to re-enable access?     

> __< plowsof >__ i can fix that for you, what is the username?     

> __< plowsof >__ (i hope i can)      

> __< l​ederstrumpf:matrix.org >__ thanks!     

> __< o​frnxmr:monero.social >__ And basicswap has bidirectional for btc and btc forks (like ltc)     

> __< l​ederstrumpf:matrix.org >__ > i can fix that for you, what is the username?     

> __< l​ederstrumpf:matrix.org >__ Lederstrumpf     

> __< o​frnxmr:monero.social >__ Post your username here after the meeting     

> __< plowsof >__ thanks will check      

> __< plowsof >__ i think we can put an open end to the meeting here, thank you all for providing feedback      

> __< h​into:monero.social >__ ﻿ofrnxmr: I think it's inappropriate to start this discussion again. I'd still be willing to write out a response detailing work, although, the CCS/updates already do this if read in full.     

> __< ofrnxmr >__ I think the subject of rates is valid     

> __< h​into:monero.social >__ plowsof: yes, all milestones are done, including binary     

> __< h​into:monero.social >__ I'll probably come back when PRs pile up, so maybe a few days from now     

> __< h​into:monero.social >__ ofrnxmr: if you believe so, then please write a comment detailing things on the proposal     

> __< o​frnxmr:monero.social >__ Its a free market, to an extent, but if you're charging 145k, and berman 250k, what Is acceptable for cryptographers?      

> __< o​frnxmr:monero.social >__ I think market rates are important, but i also think we need to be careful about setting precedent     

> __< o​frnxmr:monero.social >__ Its not _your_ proposal that's an issue, its in general     

> __< o​frnxmr:monero.social >__ people asking for funds far above what is adequate and disregarding the spirit of monero being community driven. Whales exist, but imo the source of funding isnt unlimited. What will devs do if/when whales stop contributing due to it being unsustainable? Lower their rates? Work for free? Quit?     

> __< o​frnxmr:monero.social >__ These rates being charged need to be humbled a bit. I dont know where in the world one would need over 10k/mth to live comfortably in 2024. Wen lambo     

> __< vthor >__ comfortably? I think that is absolute POV... And IMO this is a very socialist POV... And even if he would need a lambo to feel comfortably... That is his life and he decides for what he is willing to but his lifetime (the most scare in this world) into...     

> __< ofrnxmr >__ Like i said, its a free market (to an extent)     

> __< ofrnxmr >__ But ccs is supposed to support monero first and foremost      

> __< vthor >__ just my two picos....     

> __< ofrnxmr >__ If it was a free for all, then devs should open at 500k and see how long that lasts b4 we stop getting funding      

> __< ofrnxmr >__ ccs is a tool to support monero development, not a tool to buy lambos bcuz u want to overbill     

> __< vthor >__ This is something what I don't understand...     

> __< ofrnxmr >__ once upon a time there was a monero tesla ccs     

> __< vthor >__ when the CCS is aprooved it still needs to get funded, and nobody would drop money on what the person thinks it's not worth it?     

> __< p​lowsof:matrix.org >__ adds to notes 'suggest lambo experience / test drive at monerokon '     

> __< ofrnxmr >__ a lot of donations are made without reading the proposals or paying attention to the work compelted     

> __< ofrnxmr >__ We had a dev for 2 years who never fixed a single bug     

> __< C​ryptoGuard:matrix.org >__ Yes, can confirm indeed. We've looked at it and can take it on 👍     

> __< vthor >__ I calculated yesterday night ruffly estimating, that I'm working at the moment for ~$10, the lowest in my life I think.     

> __< ofrnxmr >__ underpricing is a major issue as well     

> __< ofrnxmr >__ But we red tape ppl into underpricing themself, and cause ccs proposals to fail as a result     

> __< vthor >__ "We had a dev for 2 years who never fixed a single bug" butt where is this related to the compensation?      

> __< ofrnxmr >__ he was charging for the work he wasnt doing     

> __< ofrnxmr >__ == being funded doesnt mean you earned your pay     

> __< vthor >__ this is something, but you say it's a free - market, but only to a price I can agree, and this is for me very socialist and wrong, I know I can't life comfortaby with EUR10k/month I lifed of of that and I had days where I had ony coins....     

> __< vthor >__ "== being funded doesnt mean you earned your pay" ack     

> __< ofrnxmr >__ did u also have a gambling problem 😅     

> __< vthor >__ And know things are worse, when I see inflation, I have a child with special need, the service dog, you can normally calculate daily alone $50, traveling with a dog an a cat add per night a $50 if you stay at a hotel     

> __< vthor >__ I never gambled once in my life. I make maybe unwise decisions, but at the end how I live confortable is up to me, or? Not? This is like saying nobody needs a car. Nobody needs a bicyle. Nobody needs shoes. Need != comfortable.     

> __< vthor >__ For some people it is comfortable to live in a city, for other (like for me) it is a nightmare... There are so many variables in life.... I lived of EUR50/month for some time (when I was alone), today I need almost that for purely survival of the familly a day...     

> __< ofrnxmr >__ Thats only 1500/mth     

> __< vthor >__ Don't get me wrong, it's not about attacking you or drama, it's just my view and what makes this world simply terrible, and on thing is this socialist mindset the race to the bottom....     

> __< ofrnxmr >__ Living comfortably could = drug addiction     

> __< ofrnxmr >__ i'm far from socialist     

> __< vthor >__ surveilal  != comfortable. At all....     

> __< ofrnxmr >__ But i dont like ppl pissing in the pool     

> __< vthor >__ :D     

> __< vthor >__ Well, not nice pissing in the pool, and less pissing on someones leg...     

> __< ofrnxmr >__ even worse, pissing in the lemonade      

> __< vthor >__ I see here only how here often is a push for <$50/h, and I charged normally $100/h and think this is stil very low, when I started 20 years ago IBM hat a rate >$200/h...     

> __< ofrnxmr >__ What happened to ibm job     

> __< vthor >__ My mother got mad with me because I charged my EUR60 in the begining, while she got effectivly EUR5/h, and for me that is the same. Nobody sees that you charge one hour, work 2-3, and read 10 hours for it, everybody sees always only it is too much, because it is much more...     

> __< vthor >__ I never worked for IBM, it was only what IBM charged while I was charging EUR60/h....     

> __< ofrnxmr >__ ibm is a corporation with a lot if waste. Of course they charge 4x what they pay their employees     

> __< ofrnxmr >__ Anyway, my point is that, if monero devs started tk charge 200/hr, it wouldnt take long for the well to dry up     

> __< vthor >__ But anyway, this is not the point the point is that somebody decides what is a comfortable life for somebody else, that is what is in my view communist... (and not free market) and there is no free market "to an extent". Or there is a free market or not, if not it's socialism like all over the world what is causing gov with the big dildo waiting for everybody...     

> __< ofrnxmr >__ I'm not deciding what is comfortable. i am talking about cost of living. If someone wants lambo money, they should replace sarang     

> __< vthor >__ what means sarang?     

> __< ofrnxmr >__ sarand noether, one of our finest mathematician (who left)     

> __< ofrnxmr >__ sarang*     

> __< vthor >__ Why did he leave?     

> __< o​frnxmr:monero.social >__ Diego Salazar:     

> __< ofrnxmr >__ Long story, but monero didn't treat him well     

> __< vthor >__ hm... that would me make think....     

> __< ofrnxmr >__ we pay scammers all the time, while making it difficult for good workers to even get paid. triptych wa closed unpaid iirc     

> __< vthor >__ But anyway, how I said, was not meant to be an attack, I only a bit allergic of all socialist, and it irritates me more when I see it in connection to monero, because I assume that people surrounding monero are intrested in freedom - at least that I hope...     

> __< ofrnxmr >__ i am interested in freedom as it relates to monero, not interested in ppl coming to monero to drain the well and leave      

> __< d​iego:cypherstack.com >__ Hi     

> __< ofrnxmr >__ can you explain to vthor a short version of why sarang left      

> __< d​iego:cypherstack.com >__ The noether's were our researchers several years ago. They're some of the best in the biz. They were funded via quarterly CCS requests.     

> __< vthor >__ but ofrnxmr, then is not the price the issue but the scammers or to enable the scammers. But that is what I observe when I'm reading here, best nobody get paid anything and works forever... And when you make statements like that I think also, maybe not the best place to be, seems pretty unwelcomming...     

> __< d​iego:cypherstack.com >__ They left for a variety of reasons. Firstly, raising money for your salary quarterly is very stressful. It's much less stable than a regular job. And Monero is volatile.     

> __< d​iego:cypherstack.com >__ The tax stuff becomes more than a slight annoyance when it's also your livelihood.     

> __< vthor >__ Diego. Yes, I can see that.     

> __< vthor >__ Well taxes are not only a slight annoynce, only exit is to live nowhere.     

> __< d​iego:cypherstack.com >__ Getting things like loans or apartments become very difficult as well, because those people ask about employers and potentially want to talk about employers before giving out their services.     

> __< ofrnxmr >__ One becomes a scammer by being too greedy. Like that dev i spoke about who didnt produce work buy charged $ for attending meetings that he didnt speak in      

> __< d​iego:cypherstack.com >__ Having an employer that is an online community doesn't fit any of those bills. :P     

> __< vthor >__ :D     

> __< ofrnxmr >__ Socialism is giving away free money to people just because they think they are entitled to it     

> __< d​iego:cypherstack.com >__ Lastly, they were often pushed to and fro by a community disagreeing what they should be working on. Some people wanted churn stuff. Others wanted next-gen privacy stuff. Etc.     

> __< vthor >__ yes, that is socialism, capitalism would not do that     

> __< d​iego:cypherstack.com >__ They'd been doing things for Monero for several years and after a while they both just burned out.     

> __< vthor >__ I imagine     

> __< ofrnxmr >__ what happened to shen     

> __< d​iego:cypherstack.com >__ Sarang now works for me. I own Cypher Stack (https://cypherstack.com).     

> __< d​iego:cypherstack.com >__ I have him working on monero stuff ;)     

> __< vthor >__ cool :)     

> __< d​iego:cypherstack.com >__ But he gets paid in fiat, doesn't deal with tax stuff, and his job is much more stable.     

> __< d​iego:cypherstack.com >__ So most problems resolved.     

> __< d​iego:cypherstack.com >__ Ofrnxmr shen was before my time. Exited the project as I was first coming on in fact.     

> __< ofrnxmr >__ Ok     

> __< vthor >__ I need to continue to finish m3... It's more than overdue. Have a great day together!     

> __< vthor >__ And ofrnxmr, if you have not yet done, read Atlas Shrugged or watch the 3 movies (although they don't come even close to the book), and maybe you will think about some things different afterwards.     

> __< o​frnxmr:monero.social >__ I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< o​frnxmr:monero.social >__ If blockstream or zcash pays more, its no surprise why     

> __< vthor >__ Didn't say you are a socialist (hope at least I didn't say that), I said and meant it's a socialist view...that was all I meant ;)     

> __< b​oog900:monero.social >__ The first milestone was for the design, which was mainly documented in the design issue and the initial PR.     

> __< b​oog900:monero.social >__ The first milestone was also not for 1/3 the amount of the CCS.     

> __< b​oog900:monero.social >__ binary meaning the binary RPC endpoints, not the `cuprated` binary btw     

> __< c​howbungaman:matrix.org >__ Check out Copa Monero ! Second half of Wownero V Basicswap is Streaming live now. Video and sound quality has vastly improved!!     

> __< c​howbungaman:matrix.org >__ https://www.youtube.com/live/geiGkcZYYWU?si=H3jIeNaOsrHSbj0h     

> __< c​hch3003:monero.social >__ "Near futur"... I think it will take some time. Diego will propose a new design, maybe improve content (would be a good occasion to start fresh), and then maybe I will implement it. Seems very far away.     

> __< c​hch3003:monero.social >__ "Near future"... I think it will take some time. Diego will propose a new design, maybe improve content (would be a good occasion to start fresh), and then maybe I will implement it. Seems very far away.     

> __< o​frnxmr:monero.social >__ I agree. I dont think we're _launching_ a new website within the next 4 months, even if designs were ready today     

> __< o​frnxmr:monero.social >__ We need to worry about content that ks being dropped from site, such as migrating documentation to -docs before the launch     

> __< d​iego:cypherstack.com >__ I've been learning and working on astro the past couple weeks too where I can sneak in some time.     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe there needs to be a system that makes sure funds will still properly allocated where the value justifies the expense.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe there needs to be a system that makes sure funds will still properly allocated where the value creation towards the project justifies the expense.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe there needs to be a system that makes sure funds can properly be allocated where the value creation towards the project justifies the expense even if someone asks for 50k/month.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe there needs to be a system that makes sure funds can properly be allocated where the value creation towards the project justifies the expense even if someone wanted 50k/month.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe it would good to have a system in place that consider whether or not thr value creation towards the project justifies the expense even if someone wanted 50k/month.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. Maybe it would good to have a system in place that consider whether or not the value creation towards the project justifies the expense even if someone wanted 50k/month.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. That's the beauty of capitalism to have the right to choose our rate. Maybe it would good to have a system in place that consider whether or not the value creation towards the project justifies the expense even if someone wanted 50k/month.     

> __< a​llmax:matrix.org >__ > <@ofrnxmr:monero.social> I'm not a socialist. If devs want to sink the boat with 1 million dollar ccs proposals, so be it. As a project, were not big enough to support salaries that are offered by projects with vc investors or dev fees     

> __< a​llmax:matrix.org >__ People will continuously ask for any rate they want. That's the beauty of capitalism to have the right to define our rate. Maybe it would good to have a system in place that consider whether or not the value creation towards the project justifies the expense even if someone wanted 50k/month.     

> __< o​frnxmr:monero.social >__ Of course the numbers can be justified     

> __< o​frnxmr:monero.social >__ The problem is that people are just rubber stamping due to marketing and laziness. FCMP implementation s was 300xmr for     

> __< o​frnxmr:monero.social >__ the initial FCMP ccs was for 300xmr for a couple months of work. People wanted to complain, but the work was delivered gift wrapped, and worth every piconero     

> __< o​frnxmr:monero.social >__ "What is the total estimated cost to finish cuprate"     

> __< 3​21bob321:monero.social >__ Turning into the goverment     

> __< 3​21bob321:monero.social >__ 250k or even 500k sustainable ?     

> __< 3​21bob321:monero.social >__ Just merge already, got POW and currently the only one besides you doing work on website. The rest want to nuke it and make money     

> __< s​nowman:tetaneutral.net >__ What’s the name coin endgame with regard to Monero     

> __< l​ederstrumpf:matrix.org >__ https://www.namebrow.se/name/d/atomic-trophy/     

> __< l​ederstrumpf:matrix.org >__ Registering domain names anonymously with Monero.     

> __< l​ederstrumpf:matrix.org >__ Jeremy announced their successful port at Monerokon in June: https://youtu.be/qQ4ptuh_w84?t=643.     

> __< plowsof >__ how much did zcash pay for (and on-going presumably) for their rust node? i could only find this inquiry thread https://forum.zcashcommunity.com/t/inquiry-c-to-rust-conversion/38343      

> __< plowsof >__ or if "equilibrium" would look at monero's code base / currrent cuprate and make an offer to 'finish it' that is a fraction of the cost / time of severlal full time cuprate devs we'd have something for comparison     

> __< 3​21bob321:monero.social >__ Is there a working binary ?     

> __< plowsof >__ otherwise we have a situation where i would steer people into making an alias and submitting some superficial PR's to cuprate then put up a full time mark rate ccs, rather than bothering with monero-core      

> __< plowsof >__ if their goal is monetary      

> __< 3​21bob321:monero.social >__ Also how much did it cost to make the current monerod ?     

> __< plowsof >__ hopefully we start to see funding for the stressnet workgroup to encourage contributions there too. so far cuprate and stressnet are benefiting each other though      


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)

**Comment by plowsof** [comment]
excerpt from 3rd Augustus where i missed the original meeting
```
15:04:56 <binarybaron> hey, not sure if the meeting is being postpoed but we're here to answer any questions regarding proposal anyway :)
15:05:40 <einliterflasche> https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477
15:10:17 <m-relay> <o​frnxmr:monero.social> Looks postponed to me. Of course people are still free to discuss
15:47:54 <plowsof> I missed the meeting sorry everyone. AOB?
15:53:42 <einliterflasche> Hey, not really unless there's anything blocking our proposal
```
